### PR TITLE
LPS-142404 The video preview is broken on the Item Seletor

### PR DESCRIPTION
**Pull request by boton**
BUG: https://issues.liferay.com/browse/LPS-142404

**Before the fix:**

image

**After the fix:**

image

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Carousel.es.js [hunk 1]
```diff
@@ -91,29 +91,38 @@ const Carousel = ({
...
-					src={currentItem.url || currentItem.base64}
-					title={currentItem.title}
-				/>
-			)}
-
-			{showArrows && (
-				<Arrow direction="right" handleClick={handleClickNext} />
-			)}
+}) => {
+	const isVideo = currentItem.returntype === STR_VIDEO_HTML_RETURN_TYPE;
```
Comment (AliciaGarciaGarcia) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Carousel.es.js:95]: maybe we will have to change this in the future for the shortcut item selector?

**Review by AliciaGarciaGarcia — APPROVED** [review]
I'll assume that it's ok :P

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**